AIP-81 airflowctl Include CI/breeze unit-testing and distribution commands (#48099)
mistercrunch/Airflow: dev/breeze/doc/05_test_commands.rst

  :alt: Breeze testing task-sdk-tests

Using ``breeze testing airflow-ctl-tests`` command
...............................................

> The ``breeze testing airflow-ctl-tests`` command is  allows you to run tests for Airflow CTL without
> initializing database. The Airflow CTL should not need database to be started so this acts as a
> good check to see if the Airflow CTL tests are working properly.

Reviewer: ```suggestion
The ``breeze testing airflow-ctl-tests`` command allows you to run tests for Airflow CTL without
initializing database. Airflow CTL should not require a database to start, so this acts as a
good check to see if the Airflow CTL tests are working properly.
```
